app/helpers/application_announcements_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ApplicationAnnouncementsHelper
+  def global_announcements
+    render partial: 'layouts/global_announcements'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.